src/construction/construction_utils.py

```diff
@@ -37,4 +40,65 @@ def read_construction_file(
     logger.warning(
         "Construction file not found, skipping construction..."
     )
-    return None
\ No newline at end of file
+    return None
+
+
+def _convert_formtype(formtype_value: str) -> str:
+    """Convert the formtype to a standardised format.
+
+    Args:
+        formtype_value (str): The value to standardise.
+
+    Returns:
+        str: The standardised value for formtype.
+    """
+    if pd.notnull(formtype_value):
+        if formtype_value == "1" or formtype_value == "1.0" or formtype_value == "0001":
+            return "0001"
+        elif (
+            formtype_value == "6" or formtype_value == "6.0" or formtype_value == "0006"
+        ):
+            return "0006"
+        else:
+            return None
+    else:
+        return None
+    
+
+def prepare_forms_gb(
+        snapshot_df: pd.DataFrame, 
```

Comment:
I'm wondering why it's called snapshot_df ? After we've been through staging, we always call it "full_responses_df", if we're thinking of the same thing. Thanks.

Reply:
This is what it's called when its passed into construction. I have no input on this so it might be best to ask someone else from the team.